components/CustomInput/CustomTextInput.jsx

```diff
@@ -16,20 +16,21 @@ const CustomTextInput = ({
   secureTextEntry,
   touched,
   errors,
+  style
 }) => {
   const [inputValue, setInputValue] = useDelayDebounce('', 500);
 
   useEffect(() => {
     onChangeText(inputValue);
-  }, [inputValue])
+  }, [inputValue]);
 
   return (
-    <>
+    <View style={styles.container}>
       <TextInput
         name={name}
         placeholder={placeholder}
         placeholderTextColor={theme.colors.cool_gray}
-        style={styles.input}
+        style={style || styles.input}
```

Comment:
add to default props
defaultProps
style: styles.input

Reply:
ok, done